translations/hu/get5.phrases.txt

```diff
@@ -128,6 +128,10 @@
     {
         "hu"            "Egy admin kérte ezt az időkérést, ő is fogja tudni folytatni."
```

Comment:
`Egy admin kezdeményezte ezt az időkérést, ő is fogja tudni folytatni.`

Reply:
Elfogadva a javaslat. The suggestion is accepted